casper/src/main/scala/coop/rchain/casper/reporting/ReportingCasper.scala

```diff
@@ -72,13 +68,9 @@ trait ReportingCasper[F[_]] {
 }
 
 object ReportingCasper {
-  def noop[F[_]: Sync]: ReportingCasper[F] = new ReportingCasper[F] {
-
-    override def trace(
-        block: BlockMessage
-    ): F[ReplayResult] =
+  def noop[F[_]: Sync]: ReportingCasper[F] =
+    (_: BlockMessage) =>
```

Comment:
I'm not a fan of this simplified syntax because IntelliJ will not find it when searching for implementations.